Samples/Android/Speech Devices SDK Starter App/example/attendee.properties

```diff
@@ -0,0 +1,8 @@
+#ATTENDEELIST=<User_Name@User_signature>
```

Comment:
Please rename file to participants.properties and replace Attendee content with Participant

Reply:
thanks, I rename the file as participants.properties